lib/item_stream_handler.rb

```diff
@@ -33,9 +33,25 @@ def constrain_size(arr, size)
     arr
   end
 
+  def update_tally_if_necessary
+    if Time.now.day != ItemTypeTally[:time].day
```

Comment:
Clever. It looks like the default tz in a ruby lambda is UTC, so the days will be rolling over on GMT. Is there a way to generate a `Time.now.day` that returns the current day for ET? (Accounting for DST, etc?)

Reply:
According to the Internet, all you have to do is make ENV['TZ'] = 'America/New_York'. This has been your daily reminder of why Ruby is amazing